tests/testthat/test-covariate_analysis.R

```diff
@@ -0,0 +1,64 @@
+
+test_that("FindCovariateRanges() finds ranges for continuous long data", {
+  data <- CleanData(read.csv("Contribution_continuous_long_continuous_cov.csv"))
+  all_treatments <- FindAllTreatments(data)
+  treatment_ids <- CreateTreatmentIds(all_treatments, all_treatments[1])
+  
+  wrangled_data <- ReplaceTreatmentIds(data, treatment_ids)
+  
+  ranges <- FindCovariateRanges(
+    data = wrangled_data,
+    treatment_ids = treatment_ids,
+    reference = "Paracetamol",
+    covariate_title = "covar.age"
+  )
+  
+  expected_min = c(
+    "Ibuprofen" = 98,
+    "A stiff drink" = 95,
+    "Sleep" = 97,
+    "Exercise" = NA
```

Comment:
No, because this function only checks studies that make direct comparisons to the reference.

I've added more to the roxygen comment to be more clear.